lib/resolve-config.js

```diff
@@ -38,6 +38,7 @@ export default function resolveConfig(
       : failTitle,
     failComment,
     failCommentCondition,
+    // ATTN??? We opinionated :(???
     labels: isNil(labels)
       ? ["semantic-release"]
       : labels === false
```

Comment:
So.... Regardless of configured label... We will be adding the `semantic-release` label at failure Issue creation by concatenating with already label.

We are taking the route of concatenating to avoid breaking change